src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -200,6 +195,40 @@ public PersistentDataStore() {
     return activities;
   }
 
+  public List<Setting> loadSettings()throws PersistentDataStoreException{
+      Map<Setting.SettingType, Setting> settingMap = new HashMap<>();
+      settingMap.put(Setting.SettingType.COLORS ,  new Setting(Setting.SettingType.COLORS, "white")));
+      settingMap.put(Setting.SettingType.FONT_SIZE, new Setting(Setting.SettingType.FONT_SIZE, "normal"));
+
+      List<Setting> setting = new ArrayList<>();
+    Query query = new Query("settings");
+    PreparedQuery results = datastore.prepare(query);
+
+      for (Entity entity : results.asIterable()) {
+        try {
+            Setting settings = settingMap.get(Setting.SettingType.getSettingType((String) entity.getProperty("type")));
+            settings.setValue((String) entity.getProperty("value"));
+            Setting.SettingType type = Setting.SettingType.getSettingType((String) entity.getProperty("type"));
```

Comment:
We dont need this line the lines below to create a new setting. The setting has already been created in line 200/201. We just need to update it here. Which is done in "setting.setValue()". And it is added to the settings (the settings list) in line 226.